examples/weather_station/src/app_task.cpp

```diff
@@ -378,8 +398,7 @@ void AppTask::MeasurementsTimerHandler()
 
 void AppTask::OnIdentifyStart(Identify *)
 {
-	k_timer_start(&sIdentifyTimer, K_MSEC(kIdentifyTimerIntervalMs),
-		      K_MSEC(kIdentifyTimerIntervalMs));
+	k_timer_start(&sIdentifyTimer, K_MSEC(kIdentifyTimerIntervalMs), K_MSEC(kIdentifyTimerIntervalMs));
```

Comment:
I suggest configuring IDE to format only changed lines. It's SVE branch, so I don't care, but in general it generates a lot of unnecessary conflicts.

Reply:
It is a result of my auto-formatting, and I missed that.

Reply:
I realized that it's not my auto-formatting. Restyled-io has caused these changes...